### PR TITLE
Clean up and organize everything

### PR DESCRIPTION
**Pull request by pinheadmz**
My god, it's beautiful... 

`src/warnet` ONLY contains python files (+2 json files for configuration)

Now there's specific places for everything so the kids always know where to put their toys away

```
.
├── LICENSE
├── MANIFEST.in
├── README.md
├── docs
│   ├── data.md
│   ├── developer-notes.md
│   ├── graph.md
│   ├── install.md
│   ├── lightning.md
│   ├── machines.webp
│   ├── monitoring.md
│   ├── quickrun.md
│   ├── random_internet_as_graph_n100.png
│   ├── release-process.md
│   ├── running.md
│   ├── scenarios.md
│   ├── services.md
│   └── warcli.md
├── graphs
│   └── default.graphml
├── images
│   ├── bitcoin
│   ├── rpc
│   ├── sidecar
│   └── tor
├── justfile
├── manifests
│   ├── grafana_values.yaml
│   ├── loki_values.yaml
│   ├── namespace.yaml
│   ├── rbac-config.yaml
│   ├── warnet-rpc-service.yaml
│   ├── warnet-rpc-statefulset-dev.yaml
│   └── warnet-rpc-statefulset.yaml
├── pyproject.toml
├── requirements.in
├── requirements.txt
├── scripts
│   ├── apidocs.py
│   ├── build-k8s-rpc.sh
│   ├── connect_logging.sh
│   ├── graphdocs.py
│   ├── install_logging.sh
│   └── quick_start.sh
├── src
│   ├── __init__.py
│   ├── test_framework
│   └── warnet
└── test
    ├── __init__.py
    ├── build_branch_test.py
    ├── dag_connection_test.py
    ├── data
    ├── framework_tests
    ├── graph_test.py
    ├── ln_test.py
    ├── onion_test.py
    ├── rpc_test.py
    ├── scenarios_test.py
    └── test_base.py

16 directories, 45 files
```

### CODE REVIEW

**Comment by willcl-ark** [comment]
This breaks pypi packaging entirely though :'(

**Comment by willcl-ark** [comment]
Will need something like the final commit here: https://github.com/willcl-ark/warnet/pull/17

**Comment by willcl-ark** [comment]
Cherry-picked commit over here for easier review

File: src/warnet/cli/cluster.py [hunk 1]
```diff
@@ -1,10 +1,13 @@
 import os
 import subprocess
 import sys
-from pathlib import Path
+from importlib.resources import files
 
 import click
 
+MANIFEST_PATH = files("manifests")
```
Comment (pinheadmz) [inline src/warnet/cli/cluster.py:8]: TIL, cool

**Pull request merged**